web/src/javascript/network.js

```diff
@@ -38,8 +38,11 @@ export function sendAdminRequest(url, method, data = {}) {
     body: JSON.stringify(data),
   }).then(response => response.json())
     .then(response => {
-      if(response.data !== "ok")
+      if(response.data !== "ok"){
+        if(response.error === "forbidden")
```

Comment:
The `if (response.status === 403) window.location.href = "/web/adminLogin.html";` could throw to prevent the next `then`